.storybook/preview.js

```diff
@@ -66,23 +61,9 @@ export const globalTypes = {
 const themeDecorator = (story, { globals }) => {
   const { theme } = globals;
 
-  switch (theme) {
-    case THEMES.NG_LIGHT:
-      styleElement.textContent = legacyUiStyles;
-      document.body.removeAttribute(cdsThemeAttribute);
-      document.body.style.backgroundColor = getClrUiAppBackgroundColor(theme);
-      break;
-    case THEMES.NG_DARK:
-      styleElement.textContent = legacyUiDarkStyles;
-      document.body.removeAttribute(cdsThemeAttribute);
-      document.body.style.backgroundColor = getClrUiAppBackgroundColor(theme);
-      break;
-    default:
-      styleElement.textContent = `${cdsCoreAndShimStyles.join('')}`;
-      document.body.setAttribute(cdsThemeAttribute, theme === THEMES.CORE_LIGHT ? 'light' : THEMES.CORE_DARK);
-      document.body.style.backgroundColor = null;
-      break;
-  }
+  styleElement.textContent = `${cdsCoreAndShimStyles.join('')}`;
+  document.body.setAttribute(cdsThemeAttribute, theme === THEMES.CORE_LIGHT ? 'light' : THEMES.CORE_DARK);
+  document.body.style.backgroundColor = null;
```

Comment:
I don't think this line is needed since the background color is not set at all with these change.

Reply:
Which line will be that? `document.body.style.backgroundColor = null;`

Corrected in commit fb65f0a